src/chart/bar/BarView.js

```diff
@@ -673,6 +675,7 @@ function createBackgroundShape(isHorizontalOrRadial, layout, coord) {
         return {
             x: isHorizontalOrRadial ? layout.x : coordLayout.x,
             y: isHorizontalOrRadial ? coordLayout.y : layout.y,
+            r: isHorizontalOrRadial ? layout.r : coordLayout.r,
             width: isHorizontalOrRadial ? layout.width : coordLayout.width,
```

Comment:
I think it should be `layout.rb` here. Also coordLayout is a rect. It don't have r property.

Also, I prefer not to put `radius` in the layout. See comment below.